DEVELOPMENT.md

```diff
@@ -0,0 +1,24 @@
+# Development Guide
+
+## Docker local development setup
+
+1. You should have docker installed in your system, if not click [here](https://docs.docker.com/get-docker/).
+
+2. Build the docker image:
+    ```shell
+    docker build -t clion/ubuntu/cpp-env:1.0 -f Dockerfile .
+    ```
+3. Set up CLion:
+    - Navigate to Settings > Build, Execution, Deployment > Toolchains
+    - Select _+_ then choose _Docker_ from the dropdown
+    - Click the gear icon on the right side of the server field
+    - Select the docker image  from the dropdown field
+      labeled _Container Settings_ and wait for CLion to
+      complete the configuration.
+    - Use the container setttings to mount the project root as a volume with the path `/tmp/SimpleDJ`
+		*NOTE*: It has to be an abolute path
+
+4. After configuring a Docker toolchain, you can select it in CMake profiles or in Makefile settings. Alternatively, move the toolchain to the top of the list to make it default.
```

Comment:
It was successfully built, but could not run it or debug it. Getting this error `JUCE Assertion failure in juce_linux_XWindowSystem.cpp:1718`, but the README says, it should work on GNU/Linux systems with X-Windows which my machine fulfils + I was already able to run it in the past.

Reply:
I need to investigate this. It's unclear if this is related to your using a different container in your toolchain config (see [this comment](https://github.com/kcoul/SimpleDJ/pull/2#discussion_r1254529644)) for this project. I have so say I did not run into this error when building the app so I would love to maybe follow up with you on comms to debug.